packages/prng/unix_ts_rng.go

```diff
@@ -62,12 +64,17 @@ func (utrng *UnixTimestampPrng) send() {
 	now := utrng.timeSourceFunc()
 	// reduce to last resolution
 	timePoint := now - (now % utrng.resolution)
-	//  convert to float64
+
 	buf := bytes.NewBuffer(make([]byte, 0, 8))
 	if err := binary.Write(buf, binary.LittleEndian, timePoint); err != nil {
 		panic(err)
 	}
-	pseudoR := float64(binary.BigEndian.Uint64(buf.Bytes()[:8])>>11) / (1 << 53)
+
+	// add entropy
+	h := blake2b.Sum256(buf.Bytes())
+
+	// convert to float64
+	pseudoR := float64(binary.BigEndian.Uint64(h[:8])>>11) / (1 << 53)
```

Comment:
How about `pseudoR := rand.New(rand.NewSource(timePoint)).Float64()`?
Shouldn't that have the the same effect?

Reply:
Well, not _exactly_ the same, but you know what I mean :stuck_out_tongue:

Reply:
good call, I've updated the code to do that